plugin/trino-hive/src/test/java/io/trino/plugin/hive/HiveQueryRunner.java

```diff
@@ -399,6 +406,7 @@ public static void main(String[] args)
                 .setInitialTables(TpchTable.getTables())
                 .setBaseDataDir(baseDataDir)
                 .setTpcdsCatalogEnabled(true)
+                .setTpchBucketedCatalogEnabled(false)
```

Comment:
that should be the default

Reply:
right, it is already the default, let me remove this line